test/jdk/com/sun/management/OperatingSystemMXBean/GetProcessCpuLoad.java

```diff
@@ -62,7 +66,8 @@ public static void main(String[] argv) throws Exception {
             }
         }
 
-        if (ex != null) {
+        if (good == 0 && ex != null) {
```

Comment:
The check for `ex != null` is not necessary. It should always be set if `good == 0`

Reply:
Right, it does not really need to set ex = null as as soon as there is something in "good", we pass.  But leaving the Exception hanging around seems wrong.

If good is zero then yes checking for ex is redundant.  I left the check mostly to avoid any future change breaking it, but maybe it's best to expose that and make sure we fail if good is zero.